hazelcast/src/test/java/com/hazelcast/test/starter/test/HazelcastStarterTest.java

```diff
@@ -56,4 +59,12 @@ public void testMemberWithConfig() {
         assertEquals(alwaysRunningMember.getName(), "test-name");
         alwaysRunningMember.shutdown();
     }
+
+    @Test
+    public void testGetOrCreateWorkingDir() {
+        File dir = HazelcastStarter.getOrCreateVersionDirectory("3.10-EE");
+        assertTrue(dir.exists());
```

Comment:
`assertTrue()` should have an assertion message (otherwise failures are very hard to read).